scripts/qsscheck.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+import argparse
+import fnmatch
+import os
+import os.path
+import re
+import sys
+import tinycss.css21
+import PyQt5.QtWidgets
+
+
+def get_skins(mixxx_path):
+    skins_path = os.path.join(mixxx_path, 'res', 'skins')
+    for entry in os.scandir(skins_path):
+        if entry.is_dir():
+            yield entry.name
+
+
+def make_glob(name):
+    return re.sub(r'<.*>', '*', name)
+
+
+def get_global_names(mixxx_path):
+    classnames = set()
+    objectnames = set()
+    for root, dirs, fnames in os.walk(os.path.join(mixxx_path, 'src')):
+        for fname in fnames:
+            ext = os.path.splitext(fname)[1]
+            if ext in ('.h', '.cpp'):
+                fpath = os.path.join(root, fname)
+                with open(fpath, mode='r') as f:
+                    for line in f:
+                        classnames.update(set(
+                            re.findall(r'^\s*class\s+([\w_]+)', line)))
+                        objectnames.update(set(
+                            re.findall(r'setObjectName\(.*"([^"]+)"', line)))
+            elif ext == '.ui':
+                fpath = os.path.join(root, fname)
+                with open(fpath, mode='r') as f:
+                    objectnames.update(set(
+                        re.findall(r'<widget[^>]+name="([^"]+)"', f.read())))
+    return classnames, objectnames
+
+
+def get_skin_objectnames(mixxx_path, skin):
+    skin_path = os.path.join(mixxx_path, 'res', 'skins', skin)
+    for root, dirs, fnames in os.walk(skin_path):
+        for fname in fnames:
+            if os.path.splitext(fname)[1] == '.xml':
```

Comment:
it would be nice to reduce some of the indentation in this file.  In this case, you can do:

```
if os.path.splitext(fname)[1] != '.xml':
  continue
```

And save one level